example_test.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import (
+	"fmt"
+	"github.com/DATA-DOG/godog"
+	"github.com/tranquility-bdd/tranquility"
+)
+
+var paramA, paramB string
+var resp *tranquility.Response
+var err error
+
+func iSetParamAparamBAsAnd(arg1, arg2 string) error {
+	paramA = arg1
+	paramB = arg2
+	return nil
+}
+
+func iMakeAnHttpRequestToGetPostmanecho() error {
+	var url = "https://postman-echo.com/get?foo1=" + paramA + "&foo2=" + paramB
+	var action = tranquility.Action{"GET", url, nil, nil, ""}
```

Comment:
I Think that this can be also done as ```tranquility.Action{Method: "GET", URL:url}``` that it is much easier to read.

Reply:
improved